src/containers/ProfilePage/__snapshots__/index.test.jsx.snap

```diff
@@ -185,6 +191,7 @@ exports[`renders correctly 1`] = `
                     City
                   </label>
                   <input
+                    className=""
```

Comment:
the point is that, the input has no class. when an input has no class, i should not see this line in the snapshot. i should see an empty line. it should not say className="".

Reply:
I see, that's right

Reply:
but that was the opposite, look what happened when I run the test:
![image](https://user-images.githubusercontent.com/30551053/99859008-e1061a00-2b9f-11eb-9707-481b46d92959.png)